src/app/cases/viewer/case-viewer.component.ts

```diff
@@ -95,6 +100,13 @@ export class CaseViewerComponent implements OnInit, OnDestroy {
     });
   }
 
+  buildContextMap(): ContextMap {
+    let contextMap = new ContextMap();
+    contextMap.set(Constants.CASE_REFERENCE_KEY, this.caseDetails.case_id);
+    contextMap.set(CasePaymentHistoryViewerFieldComponent.PAYMENTS_BASE_URL_KEY, this.appConfig.getPaymentsUrl());
```

Comment:
The payment URL being unique for the whole app, I don't see the benefit of passing it via the context. `AppConfig` can be injected directly inside the component.

Reply:
This has been changed now.

Reply:
What I suggested was to remove it completely from the `caseViewContext`, and use `appConfig` directly in `CasePaymentHistoryViewerFieldComponent`, see related comment on `case-payment-history-viewer-field.component.ts`